mobilizon_reshare/cli/cli.py

```diff
@@ -249,5 +250,12 @@ def publication_retry(publication_id):
     safe_execution(functools.partial(retry_publication_command, publication_id),)
 
 
+@mobilizon_reshare.command("web")
+def web():
+    uvicorn.run(
+        "mobilizon_reshare.web.backend.main:app", host="0.0.0.0", port=8000, reload=True
```

Comment:
We should make this customizable with Dynaconf and 8000 should be the default

Reply:
Also maybe the host? I'm not sure whether it matters since we support only deploying as a docker image

Reply:
I will make it configurable but for now it's just a stub.

Reply:
I made #174